nucypher/blockchain/eth/interfaces.py

```diff
@@ -95,6 +95,7 @@ class UnknownContract(InterfaceError):
 
     def __init__(self,
                  poa: bool = True,
+                 light: bool = False,
```

Comment:
I don't like having `is_light` vs `light`, but I guess we can't really avoid it with click.

Reply:
you don't like the name of the parameter or the parameter in itself?

Reply:
Just the name. I don't like how click forces us to use a different param name. :P